<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDialog</class>
 <widget class="QDialog" name="LogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/menu_icon/menu_icon/icons/database_edit.png</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Export data during logging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkWriteDuringLog">
        <property name="text">
         <string>Save data to file while logging</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QLineEdit" name="lineWriteDuringLogFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonWriteDuringLogFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Time data</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Poll time (ms)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinPollTime">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkAbsoluteTimes">
        <property name="text">
         <string>Use absolute times</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkWriteDuringLog</tabstop>
  <tabstop>lineWriteDuringLogFile</tabstop>
  <tabstop>buttonWriteDuringLogFile</tabstop>
  <tabstop>spinPollTime</tabstop>
  <tabstop>checkAbsoluteTimes</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LogDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LogDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
